src/main/java/com/minecolonies/coremod/network/messages/RecallCitizenMessage.java

```diff
@@ -78,36 +76,10 @@ public void messageOnServerThread(final RecallCitizenMessage message, final Enti
             if (building != null)
             {
                 final BlockPos loc = building.getLocation();
-
-                @Nullable final CitizenData citizenData = building.getWorker();
-                if (citizenData != null)
+                EntityCitizen citizen = building.getWorkerEntity();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'citizen' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)